blog/2024/10/this-month-in-julia-world.md

```diff
@@ -0,0 +1,56 @@
+@def title = "This Month in Julia World"
+@def authors = "Stefan Krastanov"
+@def published = "05 October 2024"
+@def rss_pubdate = Date(2024, 10, 05)
+@def rss = """Community Newsletter for September 2024"""
+
+A monthly newsletter, mostly on julia internals, digestible for casual observers. A biased, incomplete, editorialized list of what I found interesting this month, with contributions from the community.
+
+“Internals” Fora and Core Repos (Slack/Zulip/Discourse/Github):
+
+* [Well documented tips on how to minimize latency](https://docs.julialang.org/en/v1.12-dev/manual/performance-tips/#Execution-latency,-package-loading-and-package-precompiling-time) in your packages are now organized in a new section in the “Performance Tips” docs
+* The trace-compile capabilities provide a convenient way to track what the compiler is working on as you ecnounter new methods during runtime. It is extremely helpful when you want to figure out what precompile statements to add to your packages. A few quality-of-life improvements are coming up, with [better colored output for the traces](https://github.com/JuliaLang/julia/pull/55763).
+* Relatedly, recently we got a special-case of trace-compile, [the new trace-dispatch](https://github.com/JuliaLang/julia/pull/55848) which prints precompile statements only for dynamically encountered methods.
```

Comment:
This is likely to be merged soon to https://github.com/JuliaLang/julia/pull/55915

Reply:
this is really cool! And now it is included